modules/querying/pages/func/query-user-defined-functions.adoc

```diff
@@ -126,12 +131,27 @@ gadmin config apply
 
 After the parameters are successfully configured, you can access your UDFs in new queries right away.
 
+=== Upload UDFs with GSQL `PUT` command
+TigerGraph offers the ability to upload UDFs directly from GSQL.
+This allows authorized users to upload UDF files without having to access the server file system.
+
+For security reasons, this option is disabled by default and must be turned on by the TigerGraph Linux user.
```

Comment:
Replace line 138 with something like this:
"This command is securely managed by gadmin and is disabled by default, requiring a Tigergraph Linux user with gadmin access to enable it" 

Replace line 139 with something like this:
"After making modifications to your UDF files, it is recommended that you disable the ability to upload UDF files using PUT until you need to modify the files again"